lints/lint_rsa_not_null_param.go

```diff
@@ -0,0 +1,92 @@
+// lint_rsa_not_null_param.go
+/********************************************************************
```

Comment:
How is this different than the last lint? Nearly all my comments apply to this one as well.